Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -1705,4 +1705,13 @@ theorem cardinalMk_adjoin_le {E : Type u} [Field E] [Algebra F E] (s : Set E) :
 
 end IntermediateField
 
+namespace IntermediateField
```

Comment:
Why is this result here and not in `Mathlib.FieldTheory.IntermediateField.Basic`? To me that would seem the most natural choice, but if you thought about this I am all ears!

Reply:
`#find-home` suggested this place, probably because in that file it didn't import things about finite dimensional

Reply:
OK, I would honestly move it there then, perhaps around the end of the file (right before or right after the two extend/restric scalars sections) in a `section Finite`.